src/logic/notifications/notificationBuilder.js

```diff
@@ -19,15 +21,26 @@ export type NotificationsQueue = {
   afterRejection: Notification | null,
 }
 
-const standardTxNotificationsQueue: NotificationsQueue = {
-  beforeExecution: NOTIFICATIONS.SIGN_TX_MSG,
-  pendingExecution: NOTIFICATIONS.TX_PENDING_MSG,
-  afterRejection: NOTIFICATIONS.TX_REJECTED_MSG,
-  afterExecution: {
-    noMoreConfirmationsNeeded: NOTIFICATIONS.TX_EXECUTED_MSG,
-    moreConfirmationsNeeded: NOTIFICATIONS.TX_EXECUTED_MORE_CONFIRMATIONS_MSG,
-  },
-  afterExecutionError: NOTIFICATIONS.TX_FAILED_MSG,
+const customizeNotification = (notification: Notification, origin: string): Notification => {
```

Comment:
nice customization options 😄  you can only add origin.

maybe a better name would be `setOriginForNotification` or `setNotificationOrigin`

Reply:
done